jenkins/artifacts/jenkinsfile

```diff
@@ -179,7 +179,28 @@ pipeline {
             }
         }
       }
+
+      stage('Publish Nightly Build to GitHub') {
+        when {
+            expression {
+                return params.RELEASE == 'nightly' && env.BRANCH == 'main' && params.ASUP_MAKE_TARGET == 'production'
+            }
+        }
+        steps {
+           sh '''
+           targetLocation=$targetParentLocation$VERSION-$RELEASE-$BRANCH
+           wget -q -O /opt/home/gh.tar.gz "https://github.com/cli/cli/releases/download/v2.2.0/gh_2.2.0_linux_386.tar.gz"
+           tar -C /opt/home -xzf /opt/home/gh.tar.gz
+           echo $GIT_TOKEN > mytoken.txt
+           /opt/home/gh_2.2.0_linux_386/bin/gh auth login --with-token < mytoken.txt
+           /opt/home/gh_2.2.0_linux_386/bin/gh release delete nightly -y || true
+           /opt/home/gh_2.2.0_linux_386/bin/gh release create nightly $targetLocation/*.rpm $targetLocation/*.deb $targetLocation/*.gz --notes "This is harvest nightly release builds" --title "Harvest Nightly Release" --prerelease
```

Comment:
minor, replace this:
> --notes "This is harvest nightly release builds"

with:
`--notes "Harvest Nightly Release Builds"`

Reply:
done